io.github.Archeb.opentrace.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.Archeb.opentrace
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+separate-locales: false
+rename-icon: opentrace
+rename-desktop-file: opentrace.desktop
+command: OpenTrace
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Requires a PR for exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
https://github.com/flathub/flatpak-builder-lint/pull/239
Yes! PR proposed.

Reply:
but why does it need that?

Reply:
It needs superuser / cap_net_raw permission to send icmp traceroute packets, so we use `flatpak-spawn` to let it run in the host system (running in userns is not possible in theroy, https://github.com/flatpak/flatpak/issues/3858#issuecomment-702016149 )